test/index/namespaces/namespaces.snapshot.cc

```diff
@@ -78,6 +81,8 @@
   }
   
   void f(a::c_alias::E) {
```

Comment:
Would be nice to be able to point to the alias.

Reply:
Yes, I will be fixing namespace aliases later. https://github.com/sourcegraph/scip-clang/issues/109